ui/shared/status/qmldir

```diff
@@ -1,5 +1,6 @@
 StatusButton 1.0 StatusButton.qml
 StatusRadioButton 1.0 StatusRadioButton.qml
 StatusRoundButton 1.0 StatusRoundButton.qml
+StatusSectionHeadline 1.0 StatusSectionHeadline.qml
```

Comment:
BTW, I don't know why, but we no longer need to add new components to qmldir 🤷

Reply:
I think they are generated into the `.pro` file. Then they don't need to be listed in qmldir